src/capi_frontend/capi_utils.hpp

```diff
@@ -0,0 +1,28 @@
+//*****************************************************************************
+// Copyright 2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#pragma once
+#include <string>
+
+#include "../pocapi.hpp"
+#include "../precision.hpp"
+
+namespace ovms {
+class Status;
```

Comment:
Cpp does not complain about unnecessary fwd declares of Status & Inference response? 2 includes (pocapi.hpp & precision.hpp) are most likely not needed now?